### DISCUSSION TITLE
Debug Launch Error When Trying to Run a C++ Program in VSCode

### DISCUSSION THREAD

**Discussion by ikhurana21** [microsoft/vscode-cpptools#11163] [Q&A]
Hello! Whenever I try to run or debug a program (through gdb) in VSCode, I get the following error message: 

cmd /C "c:\Users\ikhur\.vscode\extensions\ms-vscode.cpptools-1.16.3-win32-x64\debugAdapters\bin\WindowsDebugLauncher.exe --stdin=Microsoft-MIEngine-In-5544q02f.ckz --stdout=Microsoft-MIEngine-Out-21arqpt2.zkv --stderr=Microsoft-MIEngine-Error-3q1wkjlb.vlh --pid=Microsoft-MIEngine-Pid-btsdtayj.grr --dbgExe=C:/MinGW/bin/gdb.exe --interpreter=mi "

Does anyone know how to fix this? I've looked around on the internet and the only suggestion has been to update PowerShell, which I have, but I keep getting the error. I also have MinGW and have changed the environment variable, and as far as I can tell, everything is up to date. Thank you!

**Comment by sean-mcmanus** [comment]
I don't think that's an error. It looks like just command line logging. What's the issue?

**Reply by Noufal-Basheer** [reply]
I have the same issue, the build works fine, but the launch fails with 
So the output is not displayed on the terminal like it is supposed to.
'Microsoft-MIEngine-Error-0sst0jkg.jbo' '--pid=Microsoft-MIEngine-Pid-35lqbphw.25l' '--dbgExe=D:/Programs/MinGW/bin/gdb.exe' '--interpreter=mi' ;9c4019e1-0b22-446a-aeb1-df1aa3a0156d' 
Any solution for this ? Thanks !
